.travis/setup-tests.sh

```diff
@@ -2,11 +2,23 @@
 
 set -eu
 
+# checkout skeleton
+git clone https://github.com/pimcore/skeleton.git /var/www
+mkdir /var/www/dev
+mkdir /var/www/dev/pimcore
+
+ln -s ~ /var/www/dev/pimcore/pimcore
+
 # add config templates
-mkdir -p var/config
-cp .travis/system.template.php var/config/system.php
-cp .travis/extensions.template.php var/config/extensions.php
-cp app/config/parameters.example.yml app/config/parameters.yml
+mkdir -p /var/www/var/config
+cp .travis/system.template.php /var/www/var/config/system.php
+cp .travis/extensions.template.php /var/www/var/config/extensions.php
+cp app/config/parameters.example.yml /var/www/app/config/parameters.yml
+
+cp .travis/composer.local.json /var/www/composer.local.json
 
 # install composer dependencies
-composer install --no-interaction --optimize-autoloader
+
+cd /var/www
+composer install --dev --no-interaction --optimize-autoloader
```

Comment:
this never install the right version of Pimcore from the current commit/branch/PR, better use something like this:

```
composer req pimcore/pimcore:dev-$TRAVIS_BRANCH#$TRAVIS_COMMIT
```